frontend/javascripts/navbar.js

```diff
@@ -388,7 +394,14 @@ function Navbar({ activeUser, isAuthenticated, isInAnnotationView, hasOrganizati
     menuItems.push(<DashboardSubMenu key="dashboard" collapse={collapseAllNavItems} />);
 
     if (isAdminOrTeamManager) {
-      menuItems.push(<AdministrationSubMenu key="admin" collapse={collapseAllNavItems} />);
+      menuItems.push(
+        <AdministrationSubMenu
+          key="admin"
+          collapse={collapseAllNavItems}
+          isAdmin={isAdmin}
+          organization={activeUser ? activeUser.organization : ""}
```

Comment:
I'd rather change the if-clause in 396 to `if (isAdminOrTeamManager && activeUser != null)`. Then, flow knows that the activeUser will exist (as it's not smart enough to infer this from the fact that the user is admin or team manager). That way, we can avoid resorting to an empty string here.

Reply:
Yes, I think that's a way nicer solution!